packages/jellyfish-api-core/__tests__/category/blockchain.test.ts

```diff
@@ -55,26 +56,103 @@ describe('masternode', () => {
     await container.stop()
   })
 
-  it('should getBlockchainInfo', async () => {
-    await waitForExpect(async () => {
+  describe('getBlockchainInfo', () => {
+    it('should getBlockchainInfo', async () => {
+      await waitForExpect(async () => {
+        const info = await client.blockchain.getBlockchainInfo()
+        await expect(info.blocks).toBeGreaterThan(1)
+      })
+
       const info = await client.blockchain.getBlockchainInfo()
-      await expect(info.blocks).toBeGreaterThan(1)
+
+      expect(info.chain).toBe('regtest')
+      expect(info.blocks).toBeGreaterThan(0)
+      expect(info.headers).toBeGreaterThan(0)
+
+      expect(info.bestblockhash.length).toBe(64)
+      expect(info.difficulty).toBeGreaterThan(0)
+      expect(info.mediantime).toBeGreaterThan(1550000000)
+
+      expect(info.verificationprogress).toBe(1)
+      expect(info.initialblockdownload).toBe(false)
+      expect(info.chainwork.length).toBe(64)
+      expect(info.size_on_disk).toBeGreaterThan(0)
+      expect(info.pruned).toBe(false)
     })
+  })
 
-    const info = await client.blockchain.getBlockchainInfo()
+  describe('getBlock', () => {
+    let blockHash: string = ''
 
-    expect(info.chain).toBe('regtest')
-    expect(info.blocks).toBeGreaterThan(0)
-    expect(info.headers).toBeGreaterThan(0)
+    beforeAll(async () => {
+      blockHash = await client.blockchain.getBlockHash(1)
+    })
 
-    expect(info.bestblockhash.length).toBe(64)
-    expect(info.difficulty).toBeGreaterThan(0)
-    expect(info.mediantime).toBeGreaterThan(1550000000)
+    it('test getblock with verbo 0, should return hash', async () => {
+      const data: string | Block<unknown> = await client.blockchain.getBlock(blockHash, 0)
+      expect(data).not.toBeNull()
+    })
 
-    expect(info.verificationprogress).toBe(1)
-    expect(info.initialblockdownload).toBe(false)
-    expect(info.chainwork.length).toBe(64)
-    expect(info.size_on_disk).toBeGreaterThan(0)
-    expect(info.pruned).toBe(false)
+    it('test getblock with verbo 1', async () => {
+      const data: string | Block<string> = await client.blockchain.getBlock<string>(blockHash, 1)
+      const block = data as Block<string>
+      expect(block).toHaveProperty('hash')
+      expect(block.confirmations).toStrictEqual(2)
+      expect(block.strippedsize).toStrictEqual(360)
+      expect(block.size).toStrictEqual(396)
+      expect(block.weight).toStrictEqual(1476)
+      expect(block.height).toStrictEqual(1)
```

Comment:
The test conditions are too strict as the masternode auto minting is time bounded to auto mint every second-ish. Currently, the way the test is written sequentially. But the test runner might not run it in that order, especially when I set to run a sub set of tests.

You can do something like this, where you test conditions 'after' and 'close to' what is supposed to be.
```ts
      expect(info.chain).toBe('regtest')
      expect(info.blocks).toBeGreaterThan(0)
      expect(info.headers).toBeGreaterThan(0)

      expect(info.bestblockhash.length).toBe(64)
      expect(info.difficulty).toBeGreaterThan(0)
      expect(info.mediantime).toBeGreaterThan(1550000000)
```

This applies for all tests, written.